src/SubMenu/index.tsx

```diff
@@ -137,6 +137,12 @@ const InternalSubMenu = (props: SubMenuProps) => {
     onTitleMouseLeave,
   );
 
+  React.useEffect(() => {
+    if (elementRef.current && activeKey === eventKey) {
+      elementRef.current.focus();
+    }
+  });
```

Comment:
Seems this code makes dead loop which makes test OOM.

Reply:
Removed that part and pushed another fix. The real issue was that the submenu item is `div` element, not `li`
![image](https://github.com/react-component/menu/assets/69504277/a5058988-b3f5-4039-b0f3-c16cf2205a53)